salt/modules/win_status.py

```diff
@@ -6,8 +6,7 @@
 
 .. versionadded:: 0.12.0
 
-:depends:   - pythoncom
```

Comment:
Note to other reviewers: `pythoncom` was removed as a dependency in this module several years ago, so this just reflects the correct state of the deps.